great_expectations/expectations/metrics/map_metric.py

```diff
@@ -620,24 +619,20 @@ def _pandas_column_map_series_and_domain_values(
 
     domain_values = df[column_name]
 
+    domain_values = domain_values[boolean_mapped_unexpected_values == True]
+    map_series = map_series[boolean_mapped_unexpected_values == True]
+
     result_format = metric_value_kwargs["result_format"]
+
     if result_format["result_format"] == "COMPLETE":
         return (
-            list(domain_values[boolean_map_unexpected_values == True]),
-            list(map_series[boolean_map_unexpected_values == True]),
+            list(domain_values),
+            list(map_series),
         )
```

Comment:
Thoughts on using `df.columns.tolist()` or `df.columns.values.tolist()`? The latter is more performant but I'm just thinking from a readability standpoint. I wasn't sure what `list(df)` did but that might just be from not having used `pandas` in a while.